client/src/outside/components/AssignmentsDialog.vue

```diff
@@ -118,6 +193,10 @@ export default {
         this.description = assignmentData.description
         this.assignmentId = assignmentData._id
         this.selectedSeed = assignmentData.seedDataId || ''
+        this.persona = assignmentData.persona
+        this.profession = assignmentData.profession.toLowerCase()
```

Comment:
Why lower case?  When this value appears in the EHR I think we need Doctor, etc.